client/package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.5.0",
+    "launch-editor": "^2.2.1",
```

Comment:
What's Launch-Editor ? & Do we need this package?

Reply:
launch-editor not needed!